apps/server/src/departments/departments.service.spec.ts

```diff
@@ -1,12 +1,13 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { DepartmentsService } from './departments.service';
+import { PrismaService } from '../prisma/prisma.service';
 
 describe('DepartmentsService', () => {
   let service: DepartmentsService;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      providers: [DepartmentsService],
```

Comment:
can we also get some tests here for the departments service? can refer to the form instances pr for some reference

Reply:
Made a new ticket. Will get to the testing later in the week hopefully.

Reply:
https://trello.com/c/rQ3mXVoA/45-departments-service-tests